pkg/network/tracer/offsetguess/tracer.go

```diff
@@ -752,8 +773,37 @@ func (t *tracerOffsetGuesser) Guess(cfg *config.Config) ([]manager.ConstantEdito
 	if err == nil && State(t.status.State) == StateReady {
 		return t.getConstantEditors(), nil
 	}
-
-	eventGenerator, err := newTracerEventGenerator(t.guessUDPv6)
+	var curNS netns.NsHandle
+	curNS, err = netns.Get()
+	if err != nil {
+		return nil, fmt.Errorf("error getting current netns: %w", err)
+	}
+	// the root NS is often 0xf0000000 which makes guessing likely to fail. If so, guess in a new NS with a different offset
+	ino, err := kernel.GetInoForNs(curNS)
+	if err != nil {
+		log.Debugf("error getting ino for current netns: %v", err)
+	}
+	if ino == 4026531840 && err != nil {
+		newNS, innerErr := netns.New()
+		if innerErr != nil {
+			return nil, fmt.Errorf("error creating offsetguess netns: %w", innerErr)
+		}
+		defer func() {
+			deferErr := newNS.Close()
```

Comment:
You could just do something like 
```go
if err := newNS.Close(); err != nil {
  log.Debugf(...)
}
```
if the intention was to use a different err variable.